source/receiver.ts

```diff
@@ -123,6 +123,9 @@ export function registerMethods(methods: Partial<MessengerMethods>): void {
   }
 
   chrome.runtime.onMessage.addListener(onMessageListener);
+
+  // Only available in the background worker
+  chrome.runtime.onMessageExternal?.addListener(onMessageListener);
```

Comment:
This alone seems to just allow all messages to flow through the extension. Next steps:

- [ ] add `{extension: 'hdlsmk34'}` target support to `sendMessage`
- [ ] add a way to mark certain methods as "externally connectable"
- [ ] disable forwarding